news/2021-04-24-new-featured-artist-grynpyret.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "New Featured Artist: Grynpyret"
+date: 2021-04-24 08:00:00 +0000
+---
+
+**Grynpyret** makes waves as our latest Featured Artist!
+
+![](https://assets.ppy.sh/artists/159/header.jpg)
+
+Ever since their own brand of eccentric melodies took root on SoundCloud in 2014, **Grynpyret** (pronounced "green parrot") has pioneering the electronic music sphere. Their sound is a blend of relaxation and peculiarity, involving a calming blend of sound samples you won't find in any other music.
```

Comment:
```suggestion
Ever since their own brand of eccentric melodies took root on SoundCloud in 2014, **Grynpyret** (pronounced "green parrot") has been pioneering the electronic music sphere. Their sound is a blend of relaxation and peculiarity, involving a calming blend of sound samples you won't find in any other music.
```